### DISCUSSION TITLE
Astrological symbols?

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#97] [Q&A]
Please have a look at the two lines from Zaborowski's treatise (https://cyfrowe.mnk.pl/publication/24392Cimelia, p. 55; seems to be
down at the moment):
![Zaborowski1635-001_square](https://user-images.githubusercontent.com/8024702/149929485-99bdda22-f068-463d-b00d-ac7a9a63a8a2.png)
![Zaborowski1635-001_Venus](https://user-images.githubusercontent.com/8024702/149929509-3ca29448-ada5-4258-a007-97e9ce06847f.png)
He discusses Parkosz's treatise (cf. #27) using two characters 
![square-2](https://user-images.githubusercontent.com/8024702/149929932-b5cba198-d9c8-4190-96f9-46196203d76a.png)![venus-1](https://user-images.githubusercontent.com/8024702/149929958-826ba06f-a010-4f13-9f13-357fa1859f4d.png)
which I interpret as 'WHITE SQUARE' U+25A1 (Unicode comment: used in astrological contexts for aspect square) and 'FEMALE SIGN' U+2640 (Venus in astrology).
Do you agree with this interpretation?

**Comment by psb1558** [comment]
I would think so, but it's hard to be sure about the square without seeing more of the context. I wonder if there's a copy of Zaborowski available somewhere else? What's the title and date?

**Reply by jsbien** [reply]
The title of the treatise is "Ortographia seu  modus recte scribendi et legendi Polonicum idioma quam utilissimus", often published as an appendix to "Rudimenta  grammatices seu octo partium orationis examen cum forma seu modo  verba exponendi". This is the case of the 1535/1536 edition quoted above. This edition is not available elsewhere, but I can upload it somewhere or just send it to you. The editions was quite numerous, but only three are available online. The text of those other editions differ slightly and does not contain the passage in question.

**Reply by psb1558** [reply]
I've sent a Dropbox file upload request to your university email. I look forward to seeing the book.

**Reply by jsbien** [reply]
Uploaded.

**Reply by psb1558** [reply]
Thanks for the file! He's explaining that a square (quadrum) is used by musicians to signal a major key (durum) while a b-like sign is used for minor (molle). A few lines down, it appears that the printer is doing his best to reproduce the "sixth vowel," which should (probably) be an o with a broken line (see below), but the closest he can come (not very close!) is the ancient symbol for Venus.

Here's the passage in a [later transcript](https://www.google.com/books/edition/Bibliografioznych_ksiag_dwoje_w_ktorych/QW5VAAAAcAAJ?hl=en&gbpv=1&dq=%22erat+autem+sexta+vocalis%22&pg=PA86&printsec=frontcover) (p. 86):

<img width="412" alt="image" src="https://user-images.githubusercontent.com/16215767/150004626-082e76b2-47b2-4f93-acd0-29fb1e7dbd50.png">

Apparently nothing to do with Astrology!

**Reply by jsbien** [reply]
Still the question is where the square came from. Do you see any other reason besides astrology to cast this type/sort?

**Reply by psb1558** [reply]
Well, clearly he's got the Venus sign in his font because of its astrological uses. I don't know much about the square. Was a shape like that really used as a musical symbol? (It doesn't look much like a modern sharp sign.) If so, it could be from a set of musical symbols. But it could equally well be from an astrological set (since the typesetter has gone to that drawer for his vowel symbol). I'd have to look at some early printed astrological books to decide. Do you have references to any of those?

**Reply by jsbien** [reply]
As for musical notation the information is available e.g. here: https://books.google.pl/books?id=TMdf1SioFk4C&lpg=PA5&ots=1KoYwu2T2j&dq=b%20rotundum%20quadratum&hl=pl&pg=PA5#v=onepage&q=b%20rotundum%20quadratum&f=false . I will look for astrological books.

**Reply by psb1558** [reply]
Alas, Google  Books decided not to display the most relevant page for me. Maybe I can get this through our library.

**Comment by jsbien** [comment]
As you quoted another edition. do you have any idea what could be uses of the o with broken line slanted backwords? The o's with the line slanted forward have been used in Polish and even recently added to Unicode.

**Reply by psb1558** [reply]
The sentence reads (I think) "The sixth vowel of the ancients was thus: ♀." It seems to be part of a theoretical intro to the tract proper, on correctly spelling the Polish language. I suspect one would turn to the ancient grammarians (Priscian, etc.) for more info, but I'm way outside my wheelhouse here.

**Reply by jsbien** [reply]
The ancient means "ancient Polish", i.e. Parkosz and the vowel is Polish specific nasal one. It's proposal was handwritten and most printers since 16th century up to now don't have types to render it correctly (in the most recent 20th century edition the characters has been written by hand and pasted into the the printed matrixes to be further used by photooffset I think). So the Venus sign is just an arbitrary replacement of the missing character. Below is the fragment of the Harvard Dictionary of Music mentioned earlier:
![HarvardDictionaryofMusic5a](https://user-images.githubusercontent.com/8024702/150142127-22ef10ad-6173-457e-94c2-18d4ed5f6ae3.png)

**Reply by psb1558** [reply]
Okay, interesting. As the edition I posted the screencap from was modern(ish), it would be interesting to see the early edition it was copied from. Too bad there aren't more editions online!

**Reply by jsbien** [reply]
The earlier editions in DjVu format are available here: https://github.com/jsbien/Zaborowski-index4djview. I can upload here the relevant fragments.
